Dockerfile.build

```diff
@@ -0,0 +1,78 @@
+FROM golang:1.7-alpine
```

Comment:
If you switch to golang:1.8-alpine you will get some caching benefits as it uses the alpine-3.5 image